data_validation/__main__.py

```diff
@@ -224,6 +241,10 @@ def build_config_managers_from_args(args):
         config_type = consts.ROW_VALIDATION
     elif validate_cmd == "Custom-query":
         config_type = consts.CUSTOM_QUERY
+        if args.tables_list is not None:
```

Comment:
I would move this down to line 284 to keep the tables_list logic near each other. 
```
if args.tables_list and config_type == consts.CUSTOM_QUERY: 
   logging.warning(
                "Tables-list/tbls flag is not supported for custom-query validations, supplied tables list will be ignored."
            )
```

Reply:
Fixed this [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/607/commits/d39c63d2ab626b5368dedafa5169876d37c4baf0)